_entries/02-01 challenge1.md

```diff
@@ -95,7 +81,7 @@ az group create --name <resource-group> --location <region>
     --location <region> \
     --kubernetes-version $version \
     --generate-ssh-keys \
-    --enable-vmss \
+    --vm-set-type VirtualMachineScaleSets \
```

Comment:
Why is the command different from the docs? https://docs.microsoft.com/en-us/azure/aks/cluster-autoscaler#create-an-aks-cluster-and-enable-the-cluster-autoscaler

Reply:
`--enable-vmss` has been deprecated since VMSS went GA.

If you run `az aks create -h` now (with the latest preview extension), you get
```
--enable-vmss                             : To be deprecated. Use vm_set_type instead
```